Jira.md

```diff
@@ -0,0 +1,29 @@
+# MetaBrainz bug tracking
+
+MetaBrainz uses Jira to track bugs. It is hosted at https://tickets.metabrainz.org/
+
+Each main MetaBrainz project has its own project in Jira.
+
+### Working on tickets
+
+If you want to work on a ticket, you don't need to ask for permission. Just check
```

Comment:
Maybe it'd be good to mention that if something is marked "Decision Required," they should resolve any open questions first. I think this status is specific to our workflow.